implementation/jobberknoll/app/security/libargon2.ts

```diff
@@ -0,0 +1,75 @@
+// SAFETY: see ADR-003
+
+// SAFETY: even if the FFI fails, the code will error out, so the user will get a 5xx error, which is bad, but safe
+const libargon2 = Deno.dlopen(
```

Comment:
Technically, when stopping, one should call `libargon2.close()` (see [here](https://man7.org/linux/man-pages/man3/dlopen.3.html)) but I don't think its neccessary in docker images. We should keep it in mind tho.